test/e2e/e2e_test.go

```diff
@@ -128,12 +136,133 @@ func main(m *testing.M) int {
 		go e2eObserverControllers(testContext, log, globalOpts.ArtifactDir)
 		defer dumpTestMetrics(log, globalOpts.ArtifactDir)
 	}
+	defer alertSLOs()
 
 	// Everything's okay to run tests
 	log.Info("executing e2e tests", "options", globalOpts)
 	return m.Run()
 }
 
+// alertSLOs creates alert for our SLO/SLIs and log when firing.
+// TODO(alberto): have a global t.Run which runs all tests first then TestAlertSLOs.
+func alertSLOs() error {
+	slosRules := &monitoringv1.PrometheusRule{
```

Comment:
My preference would be to have these rules installed with hypershift (maybe optionally with a flag), in the same way that we install recording rules (https://github.com/openshift/hypershift/blob/main/cmd/install/assets/recordingrules/hypershift.yaml)
The yaml file imho may be an easier way to publish our SLOs for other folks to read/contribute (we can reference it from the docs)

Reply:
I agree, is the second bullet point in the PR desc.
But I can do it as part of this PR. Let me update.